app/src/main/res/layout-sw600dp-land/topic_practice_header_view.xml

```diff
@@ -22,24 +22,23 @@
 
       <TextView
         android:id="@+id/master_skills_text_view"
+        style="@style/Heading2"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:fontFamily="sans-serif-medium"
         android:text="@string/topic_practice_master_these_skills"
-        android:textColor="@color/oppiaPrimaryText"
-        android:textSize="20sp"
+        android:textAlignment="viewStart"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
       <TextView
         android:id="@+id/skills_description_text_view"
+        style="@style/Subtitle1"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
```

Comment:
layout_width and layout_height can be removed from this and other files as it is `wrap_content`

Reply:
Done.